src/utils/__tests__/get-code-from-ast.spec.ts

```diff
@@ -42,7 +42,7 @@ import a from 'a';
 
     const formatted = sortImports(sortedNodes, code, {
         parser: require("recast/parsers/typescript")
-    });
+    }, {} as any);
```

Comment:
Is the cast to `as any` here necessary since the 4th argument to this `sortImports` function is the `prettierOptions` type?

Reply:
This is just there to satisfy its typing. The expected result is using the default options